server/helpers/emailTemplates/newFollowerTemplate.js

```diff
@@ -0,0 +1,33 @@
+const newFollowerTemplate = (user, followerUserName) => `<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://fonts.googleapis.com/css?family=Hind:400,500" rel="stylesheet">
+    <style>
+      * {font-family: 'Hind';}
+      .container {width: 80%;margin: 0 auto;background: #fbfbfb;padding: 30px;}
+      .username {font-size: 1.2rem;}
+      .message{font-size: 1.2rem;}
+      .reset-btn {display: inline-block;background: #2fb5ee;padding: 10px; color: #fff !important;text-decoration: none;font-size: 1rem;}
+      .logo {width: 165px; padding-bottom: 20px; border-bottom: 1px solid #2fb5ee;}
+    </style>
+  </head>
+  <body>
+    <div class="container">
+      <img class="logo"
+        src="https://res.cloudinary.com/jesseinit/image/upload/v1547303525/Logo.png"
+        alt="logo"
+      />
+      <h3 class="username">Hi ${user.fullName},</h3>
+      <p class="message">Huraayy 🚀🚀 <a href=http://localhost:3000/users/${followerUserName}>${followerUserName}</a> just followed you.
```

Comment:
Nice work Jesse but I'll like to raise a concern. The URL uses localhost which seems static, How will it be updated when deployed to production (Heroku)?

Reply:
We are yet to have a front-end url. When we have that I'll be updated to facilitate site redirect.